Automattic/wp-calypso
# Editor Welcome Tour: Add close btn in minimized view

#### Changes proposed in this Pull Request

Adds a "close tour" btn to the minimized view, along with the tour index in the resume btn label.
Partly addresses #56168



#### Testing instructions



- Sandbox a site, checkout branch, and `yarn dev --sync` from `apps/editing-toolkit` to sync sandbox.
- Go to the editor, open the sidebar, and from there click on "Welcome Guide".
- Focus and minimize the tour: Press Esc in expanded view or click the minimize button in the panel.
- A `X` btn to dismiss the tour should be available. 
- Tour index is suffixed to the resume label.

<img width="342" alt="Screenshot 2021-09-21 at 3 05 47 PM" src="https://user-images.githubusercontent.com/1705499/134167660-51aca773-3396-4a85-ad1c-431ee3e32e89.png">

#### Notes

Without any additional styling, the new button brings some "hover" effect to the mix, which may or may not be what we want. 🤷‍♂️ 

![Kapture 2021-09-21 at 15 09 58](https://user-images.githubusercontent.com/1705499/134168541-e7e9b150-598b-4c65-8b8d-46817d1cf434.gif)

CC @poligilad-auto @razvanpapadopol
